src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -78,4 +81,9 @@ public GuiSettings getGuiSettings() {
     public void setGuiSettings(GuiSettings guiSettings) {
         model.setGuiSettings(guiSettings);
     }
+
+    @Override
+    public SortedList<Pair<Person, Reminder>> getSortedReminderList() {
+        return model.getSortedReminderList();
```

Comment:
Could we use model.getSortedReminderList() instead of implementing the same method in LogicManager?

Reply:
MainWindow only accesses logic component so thats why theres this getter method

Reply:
Oh makes sense. LGTM!!! 💯